cp-ksqldb-server/include/etc/confluent/docker/ensure

```diff
@@ -40,3 +40,32 @@ then
         "${KSQL_CUB_KAFKA_TIMEOUT:-180}" \
         -b "${BOOTSTRAP_SERVER_CONFIG}"
 fi
+
+# KSQL_CONNECT_GROUP_ID is a required configuration for running embedded connect
+if ! [[ -z "${KSQL_CONNECT_GROUP_ID:=}" ]]; then
+  dub ensure KSQL_CONNECT_BOOTSTRAP_SERVERS
+  dub ensure KSQL_CONNECT_GROUP_ID
+  dub ensure KSQL_CONNECT_CONFIG_STORAGE_TOPIC
+  dub ensure KSQL_CONNECT_OFFSET_STORAGE_TOPIC
+  dub ensure KSQL_CONNECT_STATUS_STORAGE_TOPIC
+  dub ensure KSQL_CONNECT_KEY_CONVERTER
+  dub ensure KSQL_CONNECT_VALUE_CONVERTER
+  # This is required to avoid config bugs. You should set this to a value that is
+  # resolvable by all containers.
+  dub ensure KSQL_CONNECT_REST_ADVERTISED_HOST_NAME
+
+  # Default to 8083, which matches the mesos-overrides. This is here in case we extend the containers to remove the mesos overrides.
+  if [ -z "$KSQL_CONNECT_REST_PORT" ]; then
+    export KSQL_CONNECT_REST_PORT=8083
```

Comment:
If you remove `KSQL_CONNECT_REST_PORT: "8083"` from the docker-compose file you were using to test this change, is this env variable exported here actually picked up in the launch script? Wouldn't you run into the same issue I found here? https://github.com/confluentinc/ksql-images/pull/43

Reply:
good catch! thanks @stevenpyzhang